src/incubator/TransitionView/useAnimatedTransition.ts

```diff
@@ -2,7 +2,7 @@
 import {useEffect, useCallback} from 'react';
 import {runOnJS, useSharedValue, withSpring, withTiming} from 'react-native-reanimated';
 import {HiddenLocation} from '../hooks/useHiddenLocation';
-import {PanningDirections, PanningDirectionsEnum} from '../panView';
+import {PanningDirections, PanningDirectionsEnum, DEFAULT_ANIMATION_CONFIG} from '../panView';
```

Comment:
We've might already discuss this...
Don't you think It's a little weird that this hook that sits under TransitionView, takes stuff from another component?
How coupled are these two components? Isn't it better in this case to duplicate the type/configs?

Reply:
From what I remember we've talked about the props, and that they are exported in another name, so they are the same values but independent if we want to migrate only one of them.
If you mean `DEFAULT_ANIMATION_CONFIG`, then it's not a real dependency at it can be changed (it is also only internal usage so I'm not sure if it's that important).